PendingReleaseNotes

```diff
@@ -81,6 +81,9 @@
 * A health warning will now be reported if the ``require-osd-release`` flag is not
   set to the appropriate release after a cluster upgrade.
 
+* LevelDB support has been removed. ``WITH_LEVELDB'' is no longer a supported
```

Comment:
nit: use `` instead of '' after WITH_LEVELDB

Reply:
Done!